### ISSUE TITLE
Game to fast 

### ISSUE THREAD

**Issue by Hanseatenschwabe** [RAFALAMAO/Rayman-Jungle-Run-Port#1]
Its great to see the game running again, cause thats one of my prime childhood games, however its way to fast. The games at least twice as fast as the original which makes some goldcoins literally unreachable, cause rayman takes longer to turn around, as in 1-5 or 1-9.

**Comment by RAFALAMAO** [comment]
> Its great to see the game running again, cause thats one of my prime childhood games, however its way to fast. The games at least twice as fast as the original which makes some goldcoins literally unreachable, cause rayman takes longer to turn around, as in 1-5 or 1-9.

Yes bro, it's an issue due display refresh rate, only set display refresh rate to 60hz :D

**Comment by sgz886** [comment]
> > Its great to see the game running again, cause thats one of my prime childhood games, however its way to fast. The games at least twice as fast as the original which makes some goldcoins literally unreachable, cause rayman takes longer to turn around, as in 1-5 or 1-9.
> 
> Yes bro, it's an issue due display refresh rate, only set display refresh rate to 60hz :D

Setting refresh rate to 60hz works on my phone ( Xiaomi 14p) , thank you

**Comment by DaffyToons** [comment]
> takes

Is it possible to change the game's refresh rate to 120hz?
At least Fiesta Run doesn't require you to change it, but is it posssible for Jungle Run at least?
Or you need to change it to 60 if you want to play it
